.github/CODEOWNERS

```diff
@@ -374,6 +374,8 @@
 /core/domain/topic*.py @aks681
 /core/storage/topic/ @aks681
 /core/templates/dev/head/components/entity-creation-services/topic-creation.service.ts @aks681
+/core/templates/dev/head/domain/topic/topic-creation-backend-api.service.ts @aks681
+/core/templates/dev/head/domain/topic/topic-creation-backend-api.service.ts @aks681
```

Comment:
Why is this file added twice here?

(Note `topic` folder is already assigned to @aks681 on line 381)

Reply:
The topic files was previously in the entity-creation-folder, in that it was raising error to add them to the code owner, then i was asked to move them to to topic in domain so i change them according to new location without seeing the above file